client/attest.go

```diff
@@ -19,6 +19,11 @@ type AttestOpts struct {
 	// attestation protocols:
 	// https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.70.4562&rep=rep1&type=pdf
 	Nonce []byte
+	// TCG Canonical Event Log to add to the attestation.
+	// Currently, we only support PCR replay for PCRs orthogonal to those in the
+	// firmware event log, where PCRs 0-9 and 14 are often measured. If the two
+	// logs overlap, server-side verification using this library may fail.
+	CanonicalEventLog []byte
```

Comment:
Naming question, should we call it CanonicalEventLog (the format) or COSEventLog (the actual content)

Reply:
I asked this question and I believe Joe wanted to separate based on format. I think mostly because the type in each TLV should give us guidance that they are COS events vs other potential events

Reply:
SGTM